Lib/System.hpp

```diff
@@ -69,10 +67,9 @@ class System {
 
   static bool fileExists(vstring fname);
 
+  // XXX
```

Comment:
Oops. This needs to go, I'll do it before merge but after you finish any changes.